apps/meteor/client/components/UserAutoCompleteMultiple/UserAutoCompleteMultipleFederated.tsx

```diff
@@ -55,33 +55,55 @@ const UserAutoCompleteMultipleFederated = ({
 		{ keepPreviousData: true },
 	);
 
-	const options = data || [];
+	const options = useMemo(() => data || [], [data]);
```

Comment:
Does this make a difference? From what I know, useQuery return is already cached...

Reply:
You are right but this was to fix a warning from eslint.